### DISCUSSION TITLE
Accident prediction from first a person camera view

### DISCUSSION THREAD

**Discussion by bhuiyanmobasshir94** [Saturn-AI/computer-vision#17] [Ideas]
Please read papers at least 5 in this week.

**Comment by hossainalmahdi** [comment]
I've already started working on it.

**Comment by bhuiyanmobasshir94** [comment]
We have to scrutinize this library - https://github.com/roboflow/supervision

**Comment by bhuiyanmobasshir94** [comment]
![image](https://github.com/Saturn-AI/Computer-Vision/assets/31741188/d6047723-7cf9-471b-bc90-52fc32dcc896)

**Comment by bhuiyanmobasshir94** [comment]
![image](https://github.com/Saturn-AI/Computer-Vision/assets/31741188/fdd2ba7c-2920-4ed3-bd94-b660c00f0f0b)

**Comment by bhuiyanmobasshir94** [comment]
https://github.com/isl-org/MiDaS

**Comment by bhuiyanmobasshir94** [comment]
- https://github.com/ultralytics/ultralytics/tree/main/examples/YOLOv8-Region-Counter
- https://docs.ultralytics.com/guides/region-counting/

**Comment by bhuiyanmobasshir94** [comment]
https://colab.research.google.com/drive/1fvgSARb9UX_JYX6bYcVWdhGOp0BBcLA2?usp=sharing

**Comment by bhuiyanmobasshir94** [comment]
https://diffmot.github.io/
